packages/babel-standalone/src/index.js

```diff
@@ -158,6 +159,8 @@ registerPlugins({
   "proposal-class-properties": require("@babel/plugin-proposal-class-properties"),
   "proposal-decorators": require("@babel/plugin-proposal-decorators"),
   "proposal-do-expressions": require("@babel/plugin-proposal-do-expressions"),
+  "proposal-export-default": require("@babel/plugin-proposal-export-default"),
```

Comment:
@Daniel15 I noticed `standalone` was including `syntax-export-extensions`, but not the actual transforms. Any reason not to?

Reply:
well one thing is that the repl uses stage-x but none of the plugins themselves since not sure who is using it that way

Reply:
@existentialism Yeah this is fine. It might have just been an oversight. I have an item on my todo list to add a script that verifies that every syntax + transform plugin in the monorepo is listed here too. Maybe I'll file a task for that and let someone else do the work 😛